tests/test_anomaly_controller.py

```diff
@@ -139,8 +141,8 @@ def get_data_source(*args, **kwargs):
     pred_series, deviation_dict = adc._detect_anomaly(
         model_name, input_data, last_date, series, subgroup, frequency
     )
-    columns = ["y", "yhat_lower", "yhat_upper", "anomaly"]
-    pred_series[columns] = pred_series[columns].apply(pd.to_numeric)
+    # columns = ["y", "yhat_lower", "yhat_upper", "anomaly"]
+    # pred_series[columns] = pred_series[columns].apply(pd.to_numeric)
```

Comment:
Can we remove these if they aren't needed?

Reply:
Done.